src/main/java/Todo.java

```diff
@@ -0,0 +1,13 @@
+public class Todo extends Task{
+
+    public Todo(String description){
+        super(description);
+    }
+    public Todo(String description, boolean isTaskDone){
+        super(description);
+    }
+    public String getDescription(){
+        return "[T]" + super.getDescription();
+    }
+
+}
```

Comment:
I like how you structure the Todo class. It follows the coding standard and it looks quite clear.